### ISSUE TITLE
progress bar ?

### ISSUE THREAD

**Issue by tobadrobert** [Microsoft/winfile#28]
Maybe add progress bar or something to inform of actual state in copy/cut files. (or %)

**Comment by craigwi** [comment]
Yes, that would be nice.

**Comment by T8z5h3** [comment]
why can't it not jest use the windows built in one (as if you dragged and dropped a file?

**Comment by NazmusLabs** [comment]
@T8z5h3 I'm guessing that's because this app handles file move operations on its own. Remember that this predates Windows explorer, which today handles these things. Now, apps can hand of these tasks to Windows Explorer, and you see the native file copy dialog with progress bar. The app would need to remove it's own system and redirect the tasks to Windows explorer.

I'm not which approach is better, really. Keep the independent file operations and improve on it with things like adding progress bars, or let the underlying OS handle it.

**Comment by T8z5h3** [comment]
for the version that is going to be close to the original keep it as is but for the other version i say use the o.s. as it has a better understanding of NTFS.

**Comment by ZanderBrown** [comment]
@T8z5h3 but then you start getting to the point of why don't you just use Windows Explorer?

**Comment by T8z5h3** [comment]
well explorer is a good app file manager brings a lot of that functionality under one window, you can have two tree's open for copying, you have full network support,full system search, printing, folder sorting and also a full view of the folder tree.

**Comment by NazmusLabs** [comment]
@T8z5h3 yeah, Windows Explorer is a broad set of feature, being a significant part of the shell, including the start menu, action center, taskbar, file copy, and File Explorer. Some file manager apps uses their own file copy system rather than using Windows Explorer because then the app gets more control of the copy operation and UI, but some uses Windows Explorer, which is why I am not sure which would be the best approach for WinFile. I see benefits and drawbacks in both approaches.
